src/resolvers/Query/postsByOrganizationConnection.ts

```diff
@@ -33,13 +33,13 @@ export const postsByOrganizationConnection: QueryResolvers["postsByOrganizationC
         pagination: true,
         page: args.skip,
         limit: args.first,
-        populate: ["organization", "likedBy", "comments", "creator"],
+        populate: ["organization", "likedBy", "creator"],
```

Comment:
Since you've implemented `creator` for `post` as well, `creator` is no longer needed here.